docs/platforms/javascript/common/session-replay/issue-types.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Replay Issues
+sidebar_order: 4300
+notSupported:
+  - javascript.cordova
+  - javascript.wasm
+  - javascript.bun
+  - javascript.deno
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
```

Comment:
I copied this from the existing files in this folder. 

Lets follow up on the right list in another PR. I suspect that they're not 'supported' because replay records the dom, what we could have for those platforms (and php, rudy, python, etc) is a link to the basic JavaScript docs and/or a dropdown like "You're using node+express, choose your frontend framework from the dropdown to get replay instructions"